cmd/cstor-pool-mgmt/controller/common/common_test.go

```diff
@@ -144,8 +144,8 @@ func TestPoolNameHandler(t *testing.T) {
 					},
 				},
 				Spec: apis.CStorVolumeReplicaSpec{
-					TargetIP: "127.0.0.1",
-					Capacity: "100MB",
+					TargetIPSpec: apis.TargetIPSpec{"127.0.0.1"},
+					CapacitySpec: apis.CapacitySpec{"100MB"},
```

Comment:
[optional comment]
The earlier way is better IMO. Can the inputs & expected output(s) be simple type, standard primitives.
I would in-fact recommend removing the apis.CstorVolumeReplicaSpec totally & abstract that in the test logic itself.